charts/gateway-helm/templates/certgen-rbac.yaml

```diff
@@ -62,3 +62,63 @@ subjects:
 - kind: ServiceAccount
   name: '{{ include "eg.fullname" . }}-certgen'
   namespace: '{{ .Release.Namespace }}'
+---
+{{- if .Values.topologyInjector.enabled }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: '{{ include "eg.fullname" . }}-certgen:{{ .Release.Namespace }}'
```

Comment:
curious why  `:` was picked over `-` ?

Reply:
🤷 just a naming convention I've used personally and seen elsewhere for RBAC resources. I can change to `-` here.

Reply:
its fine, lets use this name, I thought dots were not allowed, but if they are, this is fine